### PR TITLE
Srs in polishing

### PR DESCRIPTION
**Pull request by autra**
This PR is motivated by #66 and #68: the faq now explains what Cesium needs.

Also, this PR make the `convert` method raise a more explanatory error message if we lack srs information in pointclouds.

@oslandia please review :-)

### CODE REVIEW

**Comment by delhomer** [comment]
Looks good to me.

File: .gitignore [hunk 1]
```diff
@@ -87,3 +87,4 @@ ENV/
 
 # Rope project settings
 .ropeproject
+.pytest
```
Comment (delhomer) [inline .gitignore:90]: I see this mention to pytest. Could we write a unit test for the addressed problem?

**Review by delhomer — APPROVED** [review]

**Comment by autra** [comment]
@delhomer I added a test (and then fix it) if you want to have a look.

**Comment by delhomer** [comment]
@autra your test is perfect. You can merge!

**Comment by autra** [comment]
thanks @delhomer

**Pull request merged**